type-definitions/immutable.js.flow

```diff
@@ -561,9 +561,78 @@ declare module 'immutable' {
     flatten(shallow?: boolean): /*this*/Map<any,any>;
   }
 
-  // OrderedMaps have nothing that Maps do not have. We do not need to override constructor & other statics
-  declare class OrderedMap<K,V> extends Map<K,V> {
+  declare class OrderedMap<K,V> extends KeyedCollection<K,V> {
```

Comment:
Ah, didn't realize that. I extendedKeyedCollection because `Immutable.Map.isMap` is a function whereas `Immutable.OrderedMap.isMap` doesn't seem to be a function:

![image](https://cloud.githubusercontent.com/assets/831651/21397312/4cf93496-c771-11e6-9797-a20d8f3ab040.png)

I'll make the change for this and `OrderedSet`.